doc/_includes/footer.html

```diff
@@ -0,0 +1,8 @@
+<footer class="doc-footer-site">
+    <div class="doc-footer-site-copyright">
+        <p class="doc-copy copy-micro">
+            &copy; <time datetime="{{ site.time | date_to_xmlschema }}">{{ site.time | date: '%Y' }}</time> edX.
+            All rights reserved.
```

Comment:
We recently got updated copyright text from our legal team for the documentation. The string is "© Copyright 2016, edX Inc. and licensed under a Creative Commons Attribution-ShareAlike 4.0 International License unless otherwise specified." It looks like you're calculating the date range more dynamically, maybe replace "all rights reserved"?